src/pages/Profile.jsx

```diff
@@ -1,72 +1,80 @@
-              Патрисия Воронцова
-            </Title>
-          </p>
+  useEffect(() => {
+    getUserResult();
+  }, []);
 
-          <div className="info-wrapper">
-            Кто, на твой взляд, лучше справляется со своими задачами?
-            <br />
-            (профессионализм)
-          </div>
-          <input type="range" min="0" max="10" value="8" step="1" className="range purple" />
-
-          <div className="info-wrapper">
-            Кто чаще других вовлекается в рабочие задачи и готов помогать другим?
-            <br />
-            (вовлеченность)
-          </div>
-          <input type="range" min="0" max="10" value="4" step="1" className="range purple" />
+  async function getUserResult() {
+    setIsPollLoading(true);
+    const response = await PollService.getUserResult(pollId);
+    setResult(response);
+    setIsPollLoading(false);
+  }
+  return (
+    <div>
+      {isPollLoading ? (
+        <Space className="data-loader">
+          <Spin size="large" />
+        </Space>
+      ) : (
+        <>
+          <Title level={2} className="page-header">
+            Профиль сотрудника
+          </Title>
 
-          <div className="info-wrapper">
-            Кто более ответственно подходит к выполнению задач?
-            <br />
-            (ответственность)
-          </div>
-          <input type="range" min="0" max="10" value="6" step="1" className="range purple" />
+          <div className="profile-wrapper">
+            <div className="profile-block">
+              <div className="info-wrapper">
+                <img
+                  src={process.env.PUBLIC_URL + '/users/' + user.userId + '.svg'}
+                  alt="user"
+                  className="info-img"
+                />
+                <Title level={4} className="page-header">
+                  {user.firstName} {user.secondName}
+                </Title>
+              </div>
 
-          <div className="info-wrapper">
-            Кто скорее отзовется на помощь коллегам?
-            <br />
-            (отзывчивость)
-          </div>
-          <input type="range" min="0" max="10" value="8" step="1" className="range purple" />
+              {Object.entries(result.resultForQuestions).map((question, index) => (
+                <div>
+                  <div className="info-wrapper">
+                    {question[0]}
+                  </div>
+                  <input type="range" 
+                          min="0" 
+                          max="10" 
+                          value={question[1]} 
+                          step="1" 
+                          readOnly
+                          className="range purple" />
+                </div>
+              ))}
+            </div>
 
-          <div className="info-wrapper">
-            К чьему мнению чаще прислушиваются?
-            <br />
-            (лидерство)
+            {Object.entries(result.resultForCompetences).length > 0 ? (
```

Comment:
можно без пустого фрагмента
```suggestion
            {Object.entries(result.resultForCompetences).length > 0 && (
```
тогда : (<></>) не нужен